doc/getting-started.md

```diff
@@ -91,10 +92,12 @@ The OpenDevStack uses [Ansible](https://www.ansible.com/ "Ansible") to install a
 ## Setup your local environment
 
 ### Prepare infrastructure
+From now on we assume, you work from a Bash (Cygwin / Linux).
```

Comment:
Can we talk about "terminal" or "shell"? I think this is more appropriate than e.g. Cygwin (which does not make sense on unix) or even bash (which is not necessarily the shell used - e.g. I work with zsh).

Noting this here but there are several occurrences further down too ..